### DISCUSSION TITLE
no-nonsence defaults?

### DISCUSSION THREAD

**Discussion by koiuo** [neovide/neovide#1653] [General]
I've been looking for a graphical neovim client, and neovide caught my eye with its _"simple, no-nonsence"_ phrase.

After the installation I've been taken aback by prolonged animations. That didn't exactly fit my expectations set by the _"simple, no-nensence"_ description.

neovide looks like a great product, starts fast, has an active community, and being capable of rendering smooth animations demonstrates rich rendering potential. 

But maybe it should be shipped with the defaults that resemble tha behavior of nvim running in a terminal (to me this as close to _simple_ as it gets)?

**Comment by MultisampledNight** [comment]
Thanks for bringing this up!

Apart from the cursor animations the only features Neovide has are blurred floating windows/popup menus, scrolling animations and ligatures. Technically those are also all nonsense, but in my opinion they all add on top of what is already provided by NeoVim and make it better.

Personally, for me the cursor animations are _the_ reason to use Neovide. This is also what [seems to be the reason for other people](https://www.reddit.com/r/neovim/comments/xnp10k/i_didnt_know_i_needed_smooth_cursor_animations/). But equally I also hear questions on how to disable the animations on Discord/Matrix, so I guess the community is on a par there.

If you feel like disabling the cursor animations, you can do so by putting this in your `init.vim`:

```vim
let g:neovide_cursor_animation_length = 0
```

(also see [the docs](https://neovide.dev/configuration.html#animation-length))

But otherwise I feel like they'll be a sensible _default_ for most users.

**Reply by koiuo** [reply]
Thank you for the response.
I've found this in the docs, no issues there.

This is just to share an opinion on what defaults should be.
If you have an understanding of what users prefer, then disregards.

Thanks for your work on the project and good luck
